tests/test_apps_setup.py

```diff
@@ -0,0 +1,15 @@
+"""
+Test the app configs for each app
+"""
+import pytest
+from django.apps import apps
+from stocks.apps import StocksConfig
+
+
+@pytest.mark.django_db(transaction=True)
+def test_stocks_apps_config():
+    """
+    Test config for stocks app
+    """
+    assert StocksConfig.name == 'stocks'
+    assert apps.get_app_config('stocks').name == 'stocks'
```

Comment:
Should we rather delete the config?

Reply:
Hmm we could do that too, I'm just following "https://stackoverflow.com/questions/43334953/testing-apps-py-in-django which doesn't seem like the most amazing source, but also what would we gain in deleting it?

Reply:
Okay, then let's ignore the lines through .coveragerc
. We don't benefit from tests like this :/